3_visualize/src/drought_learner_viz_components.R

```diff
@@ -301,64 +331,58 @@ frame_b <- function(blank_plot, streamflow_df, droughts_df,
     theme_tufte(base_family = "sans", base_size = 16)+
     theme(axis.line = element_line(color = "transparent"),
           axis.text = element_blank(),
-          axis.title = element_text(size = 5,
-                                    color = dv_tibble$dv_basePlot_axis_color),
+          axis.title = element_blank(),
           panel.background = element_blank(),
           axis.ticks = element_blank())+
     annotate("text", label = "Daily\n  streamflow", 
              x = as.Date(sprintf("13/08/%s", focal_year),'%d/%m/%Y'), hjust = 0,
-             y = 86, color = dv_tibble$dv_streamflow_textcolor_daily, size = 2)+
+             y = 86, color = dv_tibble$dv_streamflow_text_color, size = 2)+
     annotate("text", label = "Drought threshold", 
              x = as.Date(sprintf("15/08/%s", focal_year),'%d/%m/%Y'), hjust = 0,
-             y = 52, color = dv_tibble$dv_drought_textColor, size = 1.8)+
-    annotate("text", label = "Period of\ndrought", 
+             y = 52, color = dv_tibble$dv_drought_text_color, size = 1.8)+
+    annotate("text", label = "Drought", 
              x = as.Date(sprintf("23/08/%s", focal_year),'%d/%m/%Y')+0.5, hjust = 0.5,
-             y = 25, color = "#A00E00", size = 1.8)
+             y = 25, color = dv_tibble$dv_streamflow_text_color, size = 1.8)
   
-  circle <- grid::circleGrob(gp = grid::gpar(color = "#FCEE21"))
+  circle <- grid::circleGrob(gp = grid::gpar(color = dv_tibble$dv_circle_explainer))
   
   plot <- add_core_plot_elements(text = 'frame b')
   
   plot +
     draw_plot(inset,
-              x = -0.05,
-              y = 0,
-              width = 0.65,
-              height = 0.9)+
+              x = -0.1,
+              y = 0.2,
+              width = 0.6,
+              height = 0.7)+
     draw_plot(main,
-              x = 0.5,
+              x = 0.4,
               y = 0.1,
-              width = 0.5,
-              height = 0.4)  + 
+              width = 0.6,
+              height = 0.5)+
     draw_plot(zoom_plot,
-              x = 0.675,
-              y = 0.51, 
+              x = 0.61,
+              y = 0.5, 
               height = 0.4,
               width = 0.35)+
-    draw_grob(circle, scale = 0.1, y = -0.3, x = 0.353)+
-    draw_grob(circle, scale = 0.42, y = 0.24, x = 0.353)
+    draw_grob(circle, scale = 0.15, y = -0.24, x = 0.29)+
+    draw_grob(circle, scale = 0.42, y = 0.24, x = 0.29)
```

Comment:
this plotting composition could be a function itself that is fed different chart/page elements. not needed to be addressed before merging, but can work on it next.